experimental/segmenter/src/dictionary.rs

```diff
@@ -9,6 +9,14 @@ use core::str::CharIndices;
 use icu_char16trie::char16trie::{Char16Trie, TrieResult};
 use icu_provider::prelude::*;
 
+// SpacingMark and Extend for Myanmar shouldn't be first character of segment.
+//
+// TODO:
```

Comment:
Nit: Please make sure an issue is filed to track this

Reply:
Yes.